www/login.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CodeChecker</title>
+
+  <meta charset="UTF-8">
+
+  <link rel="shortcut icon" href="images/favicon.ico" />
+
+  <!-- CSS -->
+
+  <link type="text/css" rel="stylesheet" href="scripts/plugins/dojo/dijit/themes/claro/claro.css" />
+  <link type="text/css" rel="stylesheet" href="scripts/plugins/dojo/dojox/form/resources/CheckedMultiSelect.css"/>
```

Comment:
Do not import `CheckedMultiSelect`, `claroGrid`, `jsPlumb`, `marked.min`. We do not use these at the login page.